config/initializers/chartkick.rb

```diff
@@ -1,185 +1,186 @@
          marker: {
-            lineColor: '#333'
+            lineColor: "#333"
          }
       },
       boxplot: {
-         fillColor: '#505053'
+         fillColor: "#505053"
       },
       candlestick: {
-         lineColor: 'white'
+         lineColor: "white"
       },
       errorbar: {
-         color: 'white'
+         color: "white"
       }
    },
    legend: {
       itemStyle: {
-         color: '#E0E0E3'
+         color: "#E0E0E3"
       },
       itemHoverStyle: {
-         color: '#FFF'
+         color: "#FFF"
       },
       itemHiddenStyle: {
-         color: '#606063'
+         color: "#606063"
       }
    },
    credits: {
       style: {
-         color: '#666'
+         color: "#666"
       }
    },
    labels: {
       style: {
-         color: '#0088CC'
+         color: "#0088CC"
       }
    },
 
    drilldown: {
       activeAxisLabelStyle: {
-         color: '#F0F0F3'
+         color: "#F0F0F3"
       },
       activeDataLabelStyle: {
-         color: '#F0F0F3'
+         color: "#F0F0F3"
       }
    },
 
    navigation: {
       buttonOptions: {
-         symbolStroke: '#DDDDDD',
+         symbolStroke: "#DDDDDD",
          theme: {
-            fill: '#505053'
+            fill: "#505053"
          }
       }
    },
    rangeSelector: {
       buttonTheme: {
-         fill: '#505053',
-         stroke: '#000000',
+         fill: "#505053",
+         stroke: "#000000",
          style: {
-            color: '#CCC'
+            color: "#CCC"
          },
          states: {
             hover: {
-               fill: '#707073',
-               stroke: '#000000',
+               fill: "#707073",
+               stroke: "#000000",
                style: {
-                  color: 'white'
+                  color: "white"
                }
             },
             select: {
-               fill: '#000003',
-               stroke: '#000000',
+               fill: "#000003",
+               stroke: "#000000",
                style: {
-                  color: 'white'
+                  color: "white"
                }
             }
          }
       },
-      inputBoxBorderColor: '#505053',
+      inputBoxBorderColor: "#505053",
       inputStyle: {
-         backgroundColor: '#333',
-         color: 'silver'
+         backgroundColor: "#333",
+         color: "silver"
       },
       labelStyle: {
-         color: 'silver'
+         color: "#D8D8D8"
       }
    },
 
    navigator: {
       handles: {
-         backgroundColor: '#666',
-         borderColor: '#AAA'
+         backgroundColor: "#666",
+         borderColor: "#AAA"
       },
-      outlineColor: '#CCC',
-      maskFill: 'rgba(255,255,255,0.1)',
+      outlineColor: "#CCC",
+      maskFill: "rgba(255,255,255,0.1)",
       series: {
-         color: '#7798BF',
-         lineColor: '#A6C7ED'
+         color: "#7798BF",
+         lineColor: "#A6C7ED"
       },
       xAxis: {
-         gridLineColor: '#505053'
+         gridLineColor: "#505053"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.